<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-O6I509E" failures="1" tests="2" name="Test(failed)" time="0.069" errors="0" timestamp="2023-03-11T12:41:04 IST">
  <testcase classname="practiceAnnotations.Demo" name="javaTest" time="0.015">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at practiceAnnotations.Demo.javaTest(Demo.java:18)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- javaTest -->
  <testcase classname="practiceAnnotations.Demo" name="manualTest" time="0.0">
    <skipped/>
  </testcase> <!-- manualTest -->
  <testcase classname="practiceAnnotations.Demo" name="mainTest" time="0.0">
    <ignored/>
  </testcase> <!-- mainTest -->
</testsuite> <!-- Test(failed) -->
